simple_history/models.py

```diff
@@ -411,7 +411,7 @@ def copy_fields(self, model):
     def _get_history_change_reason_field(self):
         if self.history_change_reason_field:
             # User specific field from init
```

Comment:
```suggestion
            # User specific field from init.
            # It's cloned so a reason from an inherited model isn't shared.
```
Does that match the fix?

Reply:
Yeah, sure

Reply:
Yeah, sure